### DISCUSSION TITLE
Error: Invalid hook call after brand new npx create-next-app

### DISCUSSION THREAD

**Discussion by FabioBiao** [vercel/next.js#28358] [Help] [answered]
Node version: v14.15.0

NPM version: 6.14.8
Next.js v11.1.0

I decided to try next.js but i'm getting this error. 
This is after i do "next dev" after a brand new installation.
![image](https://user-images.githubusercontent.com/32443868/130287287-879755e5-1415-44cd-94e9-85507de4ec85.png)

Package version:
![image](https://user-images.githubusercontent.com/32443868/130291317-b9e49846-884f-46cb-bd48-8e8e73bd365f.png)

console errors:
![image](https://user-images.githubusercontent.com/32443868/130320400-41bbae7f-c214-4321-9359-f102dc8ffd03.png)

Do i need to install something else in order to run next.js locally? Or what am i missing?
I had the same exact error when i tried to download a next.js project from github. 

Thanks in advance.

**Comment by FabioBiao** [comment]
After some more tests...
Was able to get nextjs running by doing: 
npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter"

And after that if i do: "npm run dev" it works
If i do next dev it returns the bellow error:
![image](https://user-images.githubusercontent.com/32443868/130322773-0bef8d1b-c184-414a-9966-e351f810d291.png)


I use: npx create-next-app then if i do "npm run dev" 
i get the "Error: Cannot find module '../../../../AppData/Roaming/nvm/v14.15.0/node_modules/next/dist/shared/lib/constants.js'"
If i do the "next dev" i get the  "Error: Invalid hook call."

I tried cleaning npm cache, and deleting node_modules.
This first experience with next, haven't been great !

**Comment by FabioBiao** [comment]
After playing with the packages a lot... its working now.
not sure what caused it.

**Comment by ijjk** [comment, marked as answer]
Hi, this is due to `next` being installed globally, this can cause conflicts with packages so should only be installed in the project where it is being used and run via `yarn next` or `yarn dev`. It can be uninstalled from the global context via `npm r -g next` or `yarn global remove next`
